hedyorg/hedy
# 🪲 allow closing levels before a quiz threshold

Fixes #5046 

**How to test?**
- disable level 1 in a class
- then preview that class (or log in as a student of that class)
- you should start in level 2 with no problems.
